src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -0,0 +1,304 @@
+import * as React from 'react'
```

Comment:
similarly to FocusZone, lets create a CHANGELOG.md file in this directory to summarize what changes we did to FocusTrapZone in comparison to the original fabric version. no need to do it imediatelly but after others review and provide feedback and before we merge this PR.

Reply:
ok, will do that, once a code review will pass, before merging

Reply:
I have added information to FocusZone CHANGELOG.md